simulation-api/src/main/java/com/powsybl/simulation/securityindexes/MultiCriteriaVoltageStabilityIndex.java

```diff
@@ -73,23 +73,17 @@ public static MultiCriteriaVoltageStabilityIndex fromXml(String contingencyId, X
                 case XMLEvent.END_ELEMENT:
                     switch (xmlsr.getLocalName()) {
                         case TAG_LOCKED_TAP_CHANGER_LOAD:
-                            if (id == null) {
-                                throw new AssertionError();
-                            }
+                            Objects.requireNonNull(id);
```

Comment:
I'm not sure it's really equivalent: NullPointerException and AssertionError doesn't mean the same to me.